recipe/bld.bat

```diff
@@ -1,22 +1,36 @@
-set CONF=Release
-if "%ARCH%" == "64" (
-  set ARCH=x64
-) else (
-  set ARCH=x86
-)
+@echo on
 
-cd igraph-%PKG_VERSION%-msvc
+mkdir build
+if errorlevel 1 exit 1
+
+cd build
 
-call devenv.exe igraph.sln /Upgrade
+where cmake
+where ctest
 
-:: Build
-call msbuild.exe igraph.sln ^
-  /t:Build /v:minimal ^
-  /p:Configuration=%CONF% ^
-  /p:Platform=%ARCH%
+cmake -GNinja ^
+      -DCMAKE_BUILD_TYPE=Release ^
+      -DCMAKE_PREFIX_PATH=%CONDA_PREFIX% ^
+      -DCMAKE_INSTALL_LIBDIR=%LIBRARY_LIB% ^
+      -DCMAKE_INSTALL_INCLUDEDIR=%LIBRARY_INC% ^
+      -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -DCMAKE_POSITION_INDEPENDENT_CODE=on ^
+      -DIGRAPH_USE_INTERNAL_BLAS=0 ^
+      -DIGRAPH_USE_INTERNAL_LAPACK=0 ^
+      -DIGRAPH_USE_INTERNAL_ARPACK=1 ^
+      -DIGRAPH_USE_INTERNAL_GLPK=0 ^
+      -DIGRAPH_USE_INTERNAL_CXSPARSE=0 ^
+      -DIGRAPH_USE_INTERNAL_GMP=1 ^
```

Comment:
On windows you can use `mpir` package which is a drop in replacement for windows.